helpers/fileUpload.js

```
@@ -0,0 +1,72 @@
+const { uploader } = require('../config/cloudinaryConfig');
+const Datauri = require('datauri');
+const path = require('path');
+
+const uploadFile = async (req, res) => {
+	console.log('now here');
+	// console.log(req.files, Object.keys(req.files));
+
+	try {
+		if (Object.keys(req.files).length == 0) {
+			return res.status(400).send('No files were provided.');
```

You can as well throw an error here so the catch block can pick it up instead of repeating the logic.

---

Done sir.